### DISCUSSION TITLE
Request for Print & Export blocks programs

### DISCUSSION THREAD

**Discussion by BlueBillCanada** [pybricks/support#1378] [Ideas]
Currently printing appears to be a screenshot of everything on screen including the interface.

Feature request for printing or exporting the blocks by themselves and with the background removed at the same DPI (PNG preferred).
Also, a white background with the dots removed would be nice.

<img width="776" alt="image" src="https://github.com/pybricks/support/assets/49533129/b2a4fa59-56c2-4265-bd2c-a6bae726d0c5">

**Comment by ggramlich** [comment]
Some of your points are adressed by the new screenshot / image export function
https://github.com/pybricks/support/issues/1350
